### PR TITLE
Settings related improvements

### PR DESCRIPTION
**Pull request by cpinter**
Python usability of ctkBooleanMapper (thanks @jcfr!)

ctkComboBox supports definition of settings values for each choice through user data

### CODE REVIEW

**Comment by jcfr** [comment]
Thanks @cpinter . As soon as #741 complete and is green, I will rebase this topic. I would like to integrate it after we see a green CI

**Comment by jcfr** [comment]
I just rebased the changes.  I know expect the test to pass

File: Libs/Widgets/ctkComboBox.cpp [hunk 1]
```diff
@@ -366,3 +366,25 @@ void ctkComboBox::wheelEvent(QWheelEvent* event)
...
+
+// -------------------------------------------------------------------------
+QString ctkComboBox::currentUserDataAsString()const
+{
+  return this->itemData(this->currentIndex()).toString();
+}
+
+// -------------------------------------------------------------------------
+void ctkComboBox::setCurrentUserDataAsString(QString userData)
+{
```
Comment (jcfr) [inline Libs/Widgets/ctkComboBox.cpp]: Where do you set the provided user data ?

Comment (jcfr) [inline Libs/Widgets/ctkComboBox.cpp]: I see ... it makes sense.

**Comment by cpinter** [comment]
Thanks!

**Pull request merged**